src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -370,6 +392,7 @@ public class LibraryServiceApi implements AutoCloseable {
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final Shelf createShelf(Shelf shelf) {
+
```

Comment:
Nit: why extra lines? But no point fighting the snippet language if there isn't a good reason...